src/App.tsx

```diff
@@ -1,8 +1,97 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { PreparedTodos } from './types';
+import { TodoList } from './components/TodoList/TodoList';
+import { getPreparedTodos } from './users_&_todos_from_server_api'
```

Comment:
```suggestion
import { getPreparedTodos } from './lib/helpers'
```
зачем такое сложное название